RELEASES.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## Version 0.11.3

```

Comment:
I didn't know what to do with the versions exactly but incremented the minor at least.